### PR TITLE
Bug 1869613: Swift: add a unit test for bulk deleting failure

### PR DESCRIPTION
**Pull request by Fedosin**
This test emulates Swift behavior if it fails to delete one of the given objects in bulk mode.

### CODE REVIEW

File: pkg/storage/swift/swift_test.go [hunk 1]
```diff
@@ -335,6 +335,51 @@ func TestSwiftRemoveStorageWithContent(t *testing.T) {
...
+			w.WriteHeader(http.StatusOK)
+			respBody := `{"Number Not Found": 0, "Response Status": "200 OK", "Errors": [["obj0", "Internal Error"]], "Number Deleted": 1, "Response Body": ""}`
+			_, _ = w.Write([]byte(respBody))
+		case http.MethodDelete:
+			w.WriteHeader(http.StatusNoContent)
+			containerDeleted = true
+		}
+	})
+
+	d, installConfig := mockConfig(true, th.Endpoint()+"v3", MockUPISecretNamespaceLister{})
```
Comment (mandre) [inline pkg/storage/swift/swift_test.go:371]: `mockConfig`'s signature was changed with https://github.com/openshift/cluster-image-registry-operator/pull/596/files?file-filters%5B%5D=.go#diff-6cddc54abe4f6fba085c6fc5b982b9d9 and this call needs to be updated.

**Comment by dmage** [comment]
/hold
@Fedosin is it expected that e2e-openstack fails that often?

**Comment by Fedosin** [comment]
@dmage yes, openstack ci is highly unstable. it's not related to this PR, since it create a unit test only.

**Comment by dmage** [comment]
/hold cancel
Let's see if it manages to land.

**Comment by mandre** [comment]
e2e-openstack job was missing the optional attribute, that's the reason why prow it testing it over and over even though it's not shown as `Required` in the github UI.
Patch at https://github.com/openshift/release/pull/11521 should fix it.

**Pull request merged**